src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/clients/MessageReadinessConsumer.java

```diff
@@ -45,12 +45,12 @@ public MessageReadinessConsumer(DeadLetterQueuePeekService deadLetterQueuePeekSe
     @Override
     @Transactional
     public void run() {
-        log.debug("Running message readiness check");
+        log.info("Running message readiness check");
         try {
             final List<CaseEventMessageEntity> allMessageInNewState =
                 caseEventMessageRepository.getAllMessagesInNewState();
 
-            log.debug("Number of messages to check the readiness {}", allMessageInNewState.size());
+            log.info("Number of messages to check the readiness {}", allMessageInNewState.size());
```

Comment:
Ideally we'd be able to adjust the log level just in AAT as this will increase logging significantly.  We may need to revert this once we have the info we need.

Reply:
Hello Ravi, Yes will revert the log level later.